subsys/bluetooth/mesh/beacon.c

```diff
@@ -527,6 +541,20 @@ static bool priv_beacon_decrypt(struct bt_mesh_subnet *sub, void *cb_data)
 			params->new_key = (i > 0);
 			params->flags = out[0];
 			params->iv_index = sys_get_be32(&out[1]);
+
+#if defined(CONFIG_BT_TESTING)
+			struct bt_mesh_prb beacon_info;
+
+			STRUCT_SECTION_FOREACH(bt_mesh_beacon_cb, cb) {
```

Comment:
Missing `CONFIG_BT_MESH_PRIV_BEACONS`? Also note that mesh 1.1 shall always be able to receive private beacons. So perhaps `CONFIG_BT_MESH_V1d1` should be used instead.

Reply:
`priv_beacon_decrypt` is already guarded by `CONFIG_BT_MESH_V1d1`. But `struct bt_mesh_prb` and `priv_received` callback should be also, I agree. Changed `CONFIG_BT_MESH_PRIV_BEACONS` to `CONFIG_BT_MESH_V1d1` in `main.h`